app/src/androidTest/java/org/mifos/mobile/ApplicationTest.java

```diff
@@ -1,14 +1,15 @@
 package org.mifos.mobile;
 
-import static org.junit.Assert.assertEquals;
-
 import android.content.Context;
-import androidx.test.InstrumentationRegistry;
-import androidx.test.runner.AndroidJUnit4;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import androidx.test.InstrumentationRegistry;
```

Comment:
Why did you delete and re-write the same code? Make sure your PR contains the changes only.